engine/engine/src/test/script_props/uniform_property.script

```diff
@@ -66,8 +68,8 @@ function init(self, config)
 
     --Test setting a single element to a constant
     assert_not_error(function() go.set("#sprite", "uniform_array.y", 7, { index = 15}) end)
-    assert_number(go.get("#sprite", "uniform_array.y", { index = 17 }), 7)
```

Comment:
one of the tests which is never done. we set element 15 but try to read element 17. I fixed that